docs/docs/lib/build-your-own.mdx

```diff
@@ -574,6 +574,25 @@ There are 3 other operators:
 - `$not`
   1. Return `!evalConditionValue(conditionValue, attributeValue)`
 
+There are 6 operators that are used for comparing version strings, e.g. `v1.2.3` or `1.2.3`:
+
+| Condition | Comparison | Description                                                      |
+| --------- | ---------- | ---------------------------------------------------------------- |
+| `$veq`    | `==`       | Versions are equal                                               |
+| `$vne`    | `!=`       | Versions are not equal                                           |
+| `$vlt`    | `<`        | The first version is lesser than the second version              |
+| `$vlte`   | `<=`       | The first version is lesser than or equal to the second version  |
+| `$vgt`    | `>`        | The first version is greater than the second version             |
+| `$vgte`   | `>=`       | The first version is greater than or equal to the second version |
+
+Rules:
+
+- Segments are separated by `.` and `-` characters
+- Segments should be compared alphanumerically from the left-most segment
+  - Digit-only segments should be left-padded (e.g. with a space or a `'0'`) so that they have the same number of characters, e.g. when comparing segment `2` with `10`, the comparison should be `'02' === '10'` so that 10 is recognized as greater than 2.
```

Comment:
Padding with `0` doesn't work (that's what I had originally, but it fails when certain characters are used in the version string like `#` or `$`).  Space works because it always comes first when sorting.